pkgs/tools/misc/git-town/default.nix

```diff
@@ -1,23 +1,46 @@
-{ lib, buildGoPackage, fetchFromGitHub }:
+{ lib, buildGoModule, fetchFromGitHub, git }:
 
-buildGoPackage rec {
+buildGoModule rec {
   pname = "git-town";
   version = "7.7.0";
 
-  goPackagePath = "github.com/git-town/git-town";
   src = fetchFromGitHub {
     owner = "git-town";
     repo = "git-town";
     rev = "v${version}";
     sha256 = "sha256-FpBEBx2gb33fGDndvZmvG1A61NoJ4Qy4V3YQSb+Ugsc=";
   };
 
-  ldflags = [ "-X github.com/git-town/git-town/src/cmd.version=v${version}" "-X github.com/git-town/git-town/src/cmd.buildDate=nix" ];
+  vendorSha256 = null;
+
+  ldflags = [
+    "-s"
+    "-w"
+    "-X github.com/git-town/git-town/src/cmd.version=v${version}"
+    "-X github.com/git-town/git-town/src/cmd.buildDate=nix"
+  ];
+
+  checkInputs = [ git ];
+  preCheck =
```

Comment:
can't you use `checkFlags` for this?

Reply:
Right now, `checkFlags` is not working. See https://github.com/NixOS/nixpkgs/pull/173702.